recipes/mdbtools/recipe.yaml

```diff
@@ -0,0 +1,63 @@
+context:
+  name: mdbtools
+  version: "1.0.1"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/${{ name }}/${{ name }}/releases/download/v${{ version }}/${{ name }}-${{ version }}.tar.gz
+  sha256: "ff9c425a88bc20bf9318a332eec50b17e77896eef65a0e69415ccb4e396d1812"
+
+build:
+  number: 0
+  skip:
+    - win
+  script:
+    content: |
+      autoreconf -fis
+      ./configure --prefix="${PREFIX}"
+      make -j"${CPU_COUNT}"
+      make install
+
+requirements:
+  build:
+    - ${{ compiler('c') }}
+    - ${{ stdlib('c') }}
+    - autoconf
+    - automake
+    - gawk
+    - gettext <0.24
```

Comment:
What's the reason for this pin? In `build`, one should only add `gettext-tools` and `if: osx; then: libintl-devel` in `host`

Reply:
`gettext` ≥ 24 lacks the m4 macro definition the build needs. This leads to leftover unexpanded macros in the configure script, which does not like it. At least that's what I gathered from dissecting the arch package archives and stracing the build...

It does not use the tools, so depending on `gettext-tools` does not help, I think? It needs the m4 macro definition at compile time.